packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -70,23 +65,47 @@ function fetchingSpaceDetails() {
  * Gets details about the space
  * @param {Object} sparkInstance
  * @param {String} spaceId
+ * @param {Object} intl
  * @returns {Thunk}
  */
-export function getSpaceDetails(sparkInstance, spaceId) {
+export function getSpaceDetails(sparkInstance, spaceId, intl) {
   return (dispatch) => {
+    const {formatMessage} = intl;
     const spaceIdUUID = validateAndDecodeId(spaceId);
     if (!spaceIdUUID) {
-      const badSpaceId = `Bad Space Id`;
-      const error = new Error(badSpaceId);
-      error.name = badSpaceId;
-      return dispatch(fetchSpaceError(error));
+      const displayTitle = formatMessage(messages.unableToLoad);
+      const displaySubtitle = formatMessage(messages.badSpaceId);
+      const code = `Bad Space Id`;
+      return dispatch(addError({
```

Comment:
We're adding the errors to the store as soon as they happen instead of doing it in the selector